.github/workflows/unix.yml

```diff
@@ -206,6 +229,16 @@ jobs:
           - -DCMAKE_C_COMPILER=gcc-13
           - -DOQS_USE_OPENSSL=OFF
           - -DBUILD_SHARED_LIBS=ON -DOQS_DIST_BUILD=OFF
+        libjade-build:
+          - -DOQS_LIBJADE_BUILD=OFF
+          # Restrict -DOQS_LIBJADE_BUILD=ON build to algs provided by
+          # libjade to minimise repeated tests
+          - -DOQS_LIBJADE_BUILD=ON -DOQS_MINIMAL_BUILD=$LIBJADE_ALG_LIST
+        exclude: 
+          # macos-14 runs on aarch64, libjade targets x86
+          # Skip testing libjade on macos-14
+          - os: macos-14
```

Comment:
It seems like there is code that calls the libjade "ref" implementation on aarch64 platforms (see my comment on kem_kyber_512.c). Is this something that will actually work, or should we disable libjade completely on aarch64? (e.g., throw an error via CMake if the user passes OQS_LIBJADE_BUILD=ON on an ARM system)

Reply:
I've edited `CMakelists.txt` so that CMake explicitly disallows building liboqs with libjade on on non-x86 platforms, I'd missed this earlier. This would need to be reevaluated once libjade supports other platforms.